lib/ruby_lsp/addon.rb

```diff
@@ -182,24 +182,32 @@ def errors_details
     # reading information into memory or even spawning a separate process
     # @abstract
     #: (GlobalState, Thread::Queue) -> void
-    def activate(global_state, outgoing_queue); end
+    def activate(global_state, outgoing_queue)
+      raise AbstractMethodInvokedError
+    end
 
     # Each add-on should implement `MyAddon#deactivate` and use to perform any clean up, like shutting down a
```

Comment:
```suggestion
    # Each add-on must implement `MyAddon#deactivate` and use to perform any clean up, like shutting down a
```
If I am understanding the pattern correctly, we are now raising on abstract methods that must be implemented by the add-ons.
I suggest we update our comments to `must` instead of `should` for clarity and consistency.